src/core/templates/navigation.hbs

```diff
@@ -1,5 +1,8 @@
 {{! All links in the nav bar should have an attritbue of data-event, navigationView uses this to fire a global event}}
 <div class="navigation-inner clearfix" role="navigation" aria-label="{{_globals._accessibility._ariaLabels.navigation}}">
+    {{#if _accessibility._isSkipNavigationEnabled}}
+        <button class="skip-nav-link" data-event="skipNavigation" aria-label="{{_globals._accessibility._ariaLabels.skipNavigation}}">{{_globals._accessibility._skipNavigationText}}</button>
```

Comment:
Sorry I missed this the first time around but we usually allow HTML with triple braces `{{{` `}}}` on the actual rendered text e.g. on [Box Menu buttons](https://github.com/adaptlearning/adapt-contrib-boxmenu/blob/master/templates/boxmenu-item.hbs#L15).